Itemtree Update to 1.9

ATaylor0731 opened the pull request:
Added all items that were added in the 1.9 release. Also went ahead and fixed potions since they got changed

ATaylor0731 committed:
Fixed "Fish" issue

Changed "cooked_fished" to "cooked_fish" in ItemTree.xml to allow for
proper sorting

ATaylor0731 committed:
1.9 Items

Added all 1.9 items to ItemTree.xml

Kobata commented:
It looks like you have a general tab/space mismatch.

Kobata commented on src/main/resources/assets/inventorytweaks/ItemTree.xml:
```diff
@@ -118,13 +122,18 @@
-                <whiteStone id="end_stone"/>
+				<endStones>
+					<endStone id="end_stone"/>
+					<endStoneBricks id="end_bricks"/>
```
I'm not sure these count as 'natural' -- note the absence of stone bricks (even though they do generate in strongholds)

Kobata commented on src/main/resources/assets/inventorytweaks/ItemTree.xml:
```diff
@@ -430,6 +445,7 @@
                     <diamondBlock id="diamond_block"/>
                     <emeraldBlock id="emerald_block"/>
                     <quartzBlock id="quartz_block"/>
+					<purpurPillar id="purpur_pillar"/>
```
This isn't an ore?

ATaylor0731 replied:
I was having trouble placing the pillar block. I placed with the Quartz block since Quartz also a pillar from but it is based off DV.

ATaylor0731 committed:
Update ItemTree.xml

ATaylor0731 commented:
Update: I went through and moved Purpur pillar to the bottom of "constructionBlock", I also moved ender brick with the other "stoneBrick" I went ahead and fixed weird spacing issues that appeared when I committed.

Kobata commented:
The potion thing needs to be figured out, but if they've gone to just using NBT the solution there is probably just going to be steal the josn-NBT parser/comparison code.
